start_cerberus.py

```diff
@@ -304,6 +304,7 @@ def main(cfg):
                     logging.info("Iteration %s: Failed operators" % (iteration))
                     logging.info("%s\n" % (failed_operators))
                     dbcli.insert(datetime.now(), time.time(), 1, "degraded", failed_operators, "cluster operator")
+                    pool.map(inspect.inspect_operator, failed_operators)
```

Comment:
Think we should trigger the oc adm inspect for cluster-operators only if https://github.com/chaos-kubox/cerberus/blob/main/config/config.yaml#L22 is enabled? Thoughts?

Reply:
it should because of this line: https://github.com/paigerube14/cerberus/blob/5615226623a195371fa3edcb1cbe63959c93a1ca/cerberus/kubernetes/client.py#L282

If we dont get into the _if_ statement the failed_operators will be a blank list